src/lightning_app/runners/cloud.py

```diff
@@ -111,6 +112,11 @@ def dispatch(
             # Override the cluster ID if provided by the CLI
             app_config.cluster_id = cluster_id
 
+        if ENABLE_APP_CHECKPOINT and self.checkpoint:
+            app_config.checkpoint = self.checkpoint
```

Comment:
What is the motivation behind saving this to the app config? Are we accessing it anywhere?

Reply:
Checkpoint is a saved state to run the app anywhere from that state without updating the sourcecode. so
when running the app on the cloud, the checkpoint info needs to be sent over. Ideally, you could start the app from the UI with a different name/checkpoint which is considered an app config. 

does it make sense ?